CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## Unreleased
+
+### Changed
+
+- Package `ocramius/package-versions` replaced to internal service `Versions`
```

Comment:
### Why?

Any version of package `ocramius/package-versions` supports only 2 versions of PHP.

Using our own service, we will be able to support a wider set of PHP versions.